provider/offline_test.go

```diff
@@ -60,6 +60,13 @@ func TestOfflineStores(t *testing.T) {
 		return value
 	}
 
+	getEnv := func(key, fallback string) string {
```

Comment:
This can be replaced by [GetEnv in helpers/env.go](https://github.com/featureform/featureform/blob/main/helpers/env.go#L11-L17)